### PR TITLE
P0 authentication

### PR DESCRIPTION
**Pull request by purvasharmadev**
<h3>designed login page</h3>
 
kstyle-login-page

 <h3>designed signup page</h3>

kstyle-signup-page

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
Good to merge.

File: auth/auth.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@import url("/index.css");
+
+.input-container{
+    max-width: 500px;
+    margin:25vh auto;
```
Comment (srishtigautam21) [inline auth/auth.css:5]: you can use rem here

**Review by srishtigautam21 — APPROVED** [review]
just a small change otherwise good to merge

**Review by Omkar-Ghate — APPROVED** [review]
Good to merge.

File: auth/auth.css (hunk 1 above)
Comment (purvasharmadev) [reply auth/auth.css:5]: Did the changes. Thank you for reviewing and approving.

**Comment by purvasharmadev** [comment]
Thank you for reviewing and approving the code.

**Pull request merged**